py-rse/automate.Rmd

```diff
@@ -79,36 +72,30 @@ Make works as follows:
     if A depends on B and B depends on C—then Make will trace them through
     and run all of the commands it needs to in the right order.
 
-> **Alternatives to Make**
->
-> The first version of Make was written in 1976.
-> Programmers have created many replacements for it in the decades since then—so many,
-> in fact,
-> that none have attracted enough users to displace it.
-> If you would like to explore them,
-> check out [Snakemake][snakemake] (for Python)
-> and [drake][drake] (for R).
-> If you want to go deeper,
-> [@Smit2011] describes the design and implementation of several build tools.
-
-### Setting Up
-
 This chapter uses a version of Make called [GNU Make][gnu-make].
 It comes with macOS and Linux,
-and you can install it on Windows using [Chocolatey][chocolatey]:
+and you can install it on Windows using [Chocolatey][chocolatey] by running:
```

Comment:
Is this bit on installing make consistent with / should it refer back to the appendix where we will have all the installation instructions?

Reply:
Moved to `install.Rmd`.